config/application.rb

```diff
@@ -13,3 +13,5 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
   end
 end
+
+Rails.application.secrets.each { |key, value| ENV[key.to_s] ||= value }
```

Comment:
It's wrong way. Use please [figaro](https://github.com/laserlemon/figaro).
You should create config/application.yml and put your keys there.